packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -477,6 +521,34 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
                       canComment={canComment}
                       shouldFocusEditor={shouldFocusCommentEditor}
                     />
+                    {featureFlags.gatingEnabled && gatedGroups.length > 0 && (
```

Comment:
i suggest to remove `gatedGroups.length > 0` and [check if the current topic is gated](https://github.com/hicommonwealth/commonwealth/pull/5481/files#r1377921094), and only then show this banner.